docs/samples/specification/multiapi/generated/azure/multiapi/sample/v2/_configuration.py

```diff
@@ -28,6 +28,8 @@ class MultiapiServiceClientConfiguration(Configuration):
 
     :param credential: Credential needed for the client to connect to Azure.
     :type credential: ~azure.core.credentials.TokenCredential
+    :keyword api_version: Api Version. The default value is "2.0.0". Note that overriding this default value may result in unsupported behavior.
```

Comment:
bc we're not discriminating against single api versions in multiapi generations, we now get docstrings for api_version on the client level here